src/integration-test/java/com/esri/arcgisruntime/toolkit/BookmarksListIntegrationTest.java

```diff
@@ -0,0 +1,320 @@
+package com.esri.arcgisruntime.toolkit;
+
+import com.esri.arcgisruntime.geometry.CoordinateFormatter;
+import com.esri.arcgisruntime.geometry.GeometryEngine;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.mapping.*;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import javafx.application.Platform;
+import javafx.geometry.Insets;
+import javafx.geometry.Point2D;
+import javafx.geometry.Pos;
+import javafx.geometry.VerticalDirection;
+import javafx.scene.Scene;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.input.MouseButton;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+import javafx.util.Callback;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.testfx.api.FxRobotException;
+import org.testfx.api.FxToolkit;
+import org.testfx.framework.junit.ApplicationTest;
+import org.testfx.util.WaitForAsyncUtils;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Integration tests for BookmarksList.
+ */
+public class BookmarksListIntegrationTest extends ApplicationTest {
+
+    private StackPane stackPane;
+
+    @Override
+    public void start(Stage primaryStage) {
+        stackPane = new StackPane();
+        Scene fxScene = new Scene(stackPane);
+        primaryStage.setWidth(500);
+        primaryStage.setHeight(500);
+        primaryStage.setScene(fxScene);
+        primaryStage.show();
+        primaryStage.toFront();
+    }
+
+    @After
+    public void cleanup() throws Exception {
+        FxToolkit.cleanupStages();
+    }
+
+    /**
+     * Tests that every bookmark in a map has its name displayed in the view.
+     */
+    @Test
+    public void item_for_every_bookmark_in_map() {
```

Comment:
I'm not wedded to this naming convention. In Kotlin you can use backticks and have the test report look like natural language. Happy to change it.